index.html

```diff
@@ -160,173 +133,394 @@ <h2 class="card-head">Multi-Post <br>Stories</h2>
                 <li class="card-list-item">Front End Dev</li>
                 <li class="card-list-item">2020</li>
             </ul>
-
-            <p class="card-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups
-                required. Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
-
+            <p class="card-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
             <ul class="card-lang-list">
                 <li class="card-lang-list-item">html</li>
                 <li class="card-lang-list-item">css</li>
                 <li class="card-lang-list-item">JavaScript</li>
             </ul>
-
-            <a class="card-button-link button" href="#" rel="noopener noreferrer" target="_blank">See Project</a>
+            <button class="card-button-link button" id="see-btn4" data-bs-toggle="modal" data-bs-target="#project4">See Project</button>
         </div>
     </section>
+    
+        <!-- POPUP WINDOW - Mobile -->
+
+    <div class="modal fade " id="project1" tabindex="-1" aria-labelledby="p1-name" aria-hidden="true">
+        <div class="modal-dialog">
+          <div class="modal-content mb-content">
+            <div class="modal-header no-border-bottom">
+              <h5 class="modal-title card-head" id="p1-name">a</h5>
+              <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <ul class="card-intro-list" id="p1-description"></ul>
+                <img class="card-image" id="p1-img" src="#" alt="project-1">
+                <p class="card-paragraph" id="p1-p"></p>
+    
+                <ul class="card-lang-list modal-lang-list">
+                    <li class="card-lang-list-item" id="p1-lan1"></li>
+                    <li class="card-lang-list-item" id="p1-lan2"></li>
+                    <li class="card-lang-list-item" id="p1-lan3"></li>
+                </ul>
+                <div class="ex-margin">
+                    <a class="card-button-link button"  href="#" rel="noopener noreferrer" target="_blank">See Live <img src="images/live-btn.svg" alt="icon"></a>
+                    <a class="card-button-link button"  href="#" rel="noopener noreferrer" target="_blank">See Source <img src="images/git-btn.svg" alt="icon"></a>
```

Comment:
- [ ] Please check these buttons on popup modal. They are wrapping in screen sizes smaller than `1075px`, including mobile size and look strange 😁 

Desktop  | Mobile
------------- | -------------
<img width="331" alt="Screenshot 2022-04-15 at 16 00 16" src="https://user-images.githubusercontent.com/77038610/163568318-843bccfb-00aa-41d0-8c98-a12f5cb5ecfa.png"> | <img width="630" alt="Screenshot 2022-04-15 at 15 59 56" src="https://user-images.githubusercontent.com/77038610/163568286-30168752-44fc-42a9-a4c7-317c5f0cfe3e.png">

- [ ] In desktop size, please add adividing line between the buttons and tech tags

Your design  | Figma design
------------- | -------------
<img width="211" alt="Screenshot 2022-04-15 at 16 01 45" src="https://user-images.githubusercontent.com/77038610/163568445-6744fb2e-796a-45b6-99f5-3b4f550400e7.png">  |  <img width="249" alt="Screenshot 2022-04-15 at 16 01 20" src="https://user-images.githubusercontent.com/77038610/163568415-cfc4b328-6a5e-4321-a6e8-be654fc42788.png">

Reply:
- Modal buttons are now responsive
- A dividing line has been added after the technology tag list